Bukkit/src/main/java/com/plotsquared/bukkit/listener/BlockEventListener.java

```diff
@@ -223,48 +233,31 @@ public void onPhysicsEvent(BlockPhysicsEvent event) {
             plot.debug("Prevented block physics and resent block change because disable-physics = true");
             return;
         }
-        switch (event.getChangedType()) {
-            case COMPARATOR: {
-                if (!plot.getFlag(RedstoneFlag.class)) {
-                    event.setCancelled(true);
-                    plot.debug("Prevented comparator update because redstone = false");
-                }
-                return;
+        if (event.getChangedType() == Material.COMPARATOR) {
+            if (!plot.getFlag(RedstoneFlag.class)) {
+                event.setCancelled(true);
+                plot.debug("Prevented comparator update because redstone = false");
             }
-            case ANVIL:
-            case DRAGON_EGG:
-            case GRAVEL:
-            case SAND:
-            case TURTLE_EGG:
-            case TURTLE_HELMET:
-            case TURTLE_SPAWN_EGG: {
-                if (plot.getFlag(DisablePhysicsFlag.class)) {
+            return;
+        }
+        if (PHYSICS_BLOCKS.contains(event.getChangedType())) {
```

Comment:
this set doesn't have all the blocks that used to be in the switch?

Reply:
`ANVIL`, `DRAGON_EGG`, `GRAVEL`, and `SAND` should be covered by the `if (event.getChangedType().hasGravity() ...` above, `TURTLE_HELMET` is an item only